### PR TITLE
[EDU-805] Updates and refactoring (& Gardening)

### PR DESCRIPTION
**Pull request by fliptopbox**
## important
This PR has been extended to accommodate the changes required by EDU-805

- Replace in README all TLD `.io` with `.com`
- Replace remote assets with local assets (ie. images in the readme)
- Update the tutorial md with `.com` TLD
- Replace bitmap banner with HTML banner w. static assets logo (CSS added to index.html header)
- correct all instancing pattern with the (new) keyword

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -61,25 +61,25 @@ This information is used by the main server thread to maintain a list of active
...
 - Click on the **API Keys** tab
 - Copy the secret **API Key** value from your Root key.
 
-The server-side scripts connect to Ably using [Basic Authentication](https://www.ably.io/documentation/core-features/authentication#basic-authentication), i.e. by using the API Key directly as shown below:
+The server-side scripts connect to Ably using [Basic Authentication](https://www.ably.com/documentation/core-features/authentication#basic-authentication), i.e. by using the API Key directly as shown below:
 
 ```js
-const realtime = Ably.Realtime({
+const realtime = new Ably.Realtime({
   key: ABLY_API_KEY,
```
Comment (marcduiker) [inline TUTORIAL.md:78]: I don't see any use of the `env` file that contains the Ably key in this tutorial. Is this on purpose?

File: README.md [hunk 2]
```diff
@@ -45,43 +45,39 @@ You can check out a [blog article](https://www.ably.io/blog/a-scalable-realtime-
...
+2. Change directory to the project folder and Install dependencies
 
-   ```sh
-   cd realtime-quiz-framework
-   ```
+```sh
+npm install
 
-3. Install the dependencies
+cd realtime-quiz/
```
Comment (marcduiker) [inline README.md:57]: You could add a new script entry in package.json and combine the npm install steps using `&&`. See https://github.com/ably-labs/github-commit-visualizer/blob/baf7a1cb35091bcc063752351512ecc1e8e3e881/package.json#L15

Comment (fliptopbox) [reply README.md:57]: that is a nice idea, ATM i am using the legacy set-up instuctions, but sure we should add that `&&` catenation to the readme.

File: TUTORIAL.md (hunk 1 above)
Comment (fliptopbox) [reply TUTORIAL.md:78]: very good catch. thanks.

**Review by marcduiker — APPROVED** [review]
LGTM

**Pull request merged**